components/brave_new_tab_ui/components/search/SearchResults.tsx

```diff
@@ -9,11 +9,12 @@ import styled from 'styled-components'
 import SearchResult from './SearchResult'
 import getNTPBrowserAPI, { SearchEngineInfo } from '../../api/background'
 import { omniboxController, search, useSearchContext } from './SearchContext'
-import { braveSearchHost } from './config'
 import { stringToMojoString16 } from 'gen/ui/webui/resources/tsc/js/mojo_type_util';
 import { handleOpenURLClick, validateScheme } from '$web-common/SecureLink';
 import MaybePromptEnableSuggestions from './MaybePromptEnableSuggestions';
 
+const braveSearchHost = 'search.brave.com'
```

Comment:
I think this change should be reverted, so we have one source of truth for the `braveSearchHost`